scripts/HO-example.jl

```diff
@@ -0,0 +1,110 @@
+# # Example 5: Harmonic oscillator
+
+# `Rimu` can also handle non-lattice systems. This example shows how a harmonic 
+# oscillator Hamiltonian is implemented, and how to calculate blocks of energy 
+# eigenstates in the Hamiltonian and angular momentum eigenstates.
```

Comment:
```suggestion
# `Rimu` can also handle non-lattice systems. In this example we will look at 
# weakly-interacting bosonic particles in a harmonic oscillator external 
# potential. We will work in the basis of (Cartesian product) single-particle 
# eigenstates of the harmonic oscillator potential. We identify blocks of 
# degenerate non-interacting states that are coupled by a contact interaction 
# in first order degenerate perturbation theory. We will calculate the energy 
# eigenstates in the degenerate blocks as well as eigenstates of angular 
# momentum.
```

Reply:
This is fine, I just rewrote it to have a consistent style with the rest of the example